library/item/item_content_registry/src/main/java/org/quiltmc/qsl/item/content/registry/api/ItemContentRegistries.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.item.content.registry.api;
+
+import com.mojang.serialization.Codec;
+import org.quiltmc.qsl.registry.attachment.api.RegistryEntryAttachment;
+
+import net.minecraft.item.Item;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+/**
+ * Holds {@link RegistryEntryAttachment}s for different properties that items can hold.
+ * <p>
+ * Current properties:
+ * <ul>
+ *     <li>{@link #FUEL_TIME}</li>
+ *     <li>{@link #COMPOST_CHANCE}</li>
+ * </ul>
+ */
+public class ItemContentRegistries {
+	/**
+	 * The namespace for the content registries.
+	 */
+	public static final String NAMESPACE = "quilt_item_content_registry";
+
+	/**
+	 * A {@link RegistryEntryAttachment} for how long different items burn in a furnace. The value is stored in ticks.
+	 * <p>
+	 * Values can be set via code and through a datapack with the file {@code data/quilt_item_content_registry/attachments/minecraft/item/fuel_time.json}
+	 */
+	public static final RegistryEntryAttachment<Item, Integer> FUEL_TIME = RegistryEntryAttachment
```

Comment:
Once #128 gets merged, you should add listeners to the "entry/tag associated with value" events so the fuel/compost chance maps get updated immediately...

Reply:
1. The map that this uses is updated on datapack reloads
2. The implementation at the time of this comment (which is removed), used the attachment directly, and did not use a map inbetween

Reply:
Those "entry/tag associated with value" events are for built-in values, which can theoretically be set at any time.
(my mistake on 2 for the old impl though)

Reply:
Yeah, but if a built in value is being set when the game is running I will not be happy

Reply:
Also valid, and now I'm considering adding a "NOTE: This method should _only_ be used while registering the entry! Mods are allowed to ignore values should they be set later." note to the `put` methods...